Core1Platform/HelpAndSupport/02_CreateTicket.md

```diff
@@ -0,0 +1,51 @@
+# Create ticket
+
+The creation of a ticket is necessary, if something happens in your *Actindo Core1 Platform* that you think may be a bug in the Actindo software. In this case, the Actindo team has the chance to solve the bug and can provide an error correction to you and all other customers.   
```

Comment:
```suggestion
The creation of a ticket is necessary if something happens in your *Actindo Core1 Platform* that you think may be a bug in the Actindo software. In this case, the Actindo team has the chance to solve the bug and can provide an error correction to you and all other customers.   
```
--> Wollen wir wirklich das zugeben? ;-) Ich würde es anders/breiter formulieren, z.B.: if you are experiencing performance issues with your *Actindo Core1 Platform*... oder if you need support from our MPS team.

Reply:
you think may be a bug in the Actindo software